test/google_scraping/dashboard/keyword_scraper_test.exs

```diff
@@ -0,0 +1,23 @@
+defmodule GoogleScraping.Dashboard.KeywordScraperTest do
+  use GoogleScraping.DataCase, async: true
+
+  alias GoogleScraping.Dashboard.KeywordScraper
+
+  describe "get_search_page_html_for_keyword/1" do
+    test "given keyword, returns HTML search result" do
+      use_cassette "google/cat" do
+        {:ok, response} = KeywordScraper.get_search_page_html_for_keyword("cat")
+
+        assert response != nil
```

Comment:
I think this is returning some HTML string, right?
Would `assert is_binary(response) == true` work? 💭